packages/angular/src/generators/stories/schema.json

```diff
@@ -30,6 +30,16 @@
       "description": "Skip formatting files.",
       "type": "boolean",
       "default": false
+    },
+    "ignorePaths": {
+      "type": "string",
+      "description": "Paths to ignore when looking for components, comma-separated.",
+      "examples": [
+        "apps/my-app/src/not-stories/**",
```

Comment:
Can I also pass template strings like `${projectRoot}` and/or `${workspaceRoot}`? Like like '${projectRoot}/**/not-stories/**`

Reply:
It's not possible as it is atm. That's a great addition, I'll look on how to implement